docs/troubleshoot/minimal.md

```diff
@@ -0,0 +1,100 @@
+| Total events processed                | The number of events that were tried for forward migration, including retries and permanent errors.                              |
+| elementsReconsumedFromDeadLetterQueue | The total number of events consumed from DLQ for retry                                                                           |
+
+
+### Metrics for retryDLQ run
+
+| Metric Name                           | Description                                                                                         |
+|---------------------------------------|-----------------------------------------------------------------------------------------------------|
+| Successful events                     | Total number of events successfully processed and applied to Spanner database                       |
+| elementsReconsumedFromDeadLetterQueue | The total number of events consumed from DLQ for retry                                              |
+| Elements requeued for retry           | The total number of events that were re queued for retry                                            |
+| Conversion errors                     | Number of events that could not be converted to Spanner.This is a permanent error category.         |
+| Skipped events                        | The events that are skipped from migration since the table was dropped from migration               |
+| Other permanent errors                | The remaining permanent errors.                                                                     |
+| Total events processed                | The number of events that were tried for forward migration, including retries and permanent errors. |
+
+It can happen that in retryDLQ mode, there are still permanent errors. To identify that all the retryable errors have been processed and only permanent errors remain for reprocessing - one can look at the ‘Successful events' count - it would remain constant after every retry iteration. Each retry iteration, the ‘elementsReconsumedFromDeadLetterQueue' would increment.
+
+### Retry command
+
+This will reprocess the records marked as ‘severe' error records from the DLQ.  
+Before running the Dataflow job, check if the main Dataflow job has non-zero retryable error count. In case there are referential error records - check that the dependent table data is populated completely from the source database.
+
+Sample command to run the Dataflow job in retryDLQ mode is
+
+```sh
+gcloud beta dataflow flex-template run <jobname> --region=<the region where the dataflow job must run> --template-file-gcs-location=<location of the template image specification json> --additional-experiments=use_runner_v2 --parameters inputFilePattern=<GCS location of the input file pattern>,streamName=<Datastream name>,instanceId=<Spanner Instance Id>,databaseId=<Spanner Database Id>,sessionFilePath=<GCS path to session file>,deadLetterQueueDirectory=<GCS path to the DLQ>,runMode=retryDLQ
```

Comment:
replace the actual template in each of these commands

Reply:
Done